From 12eeef18faa0e75bcd8341a626db211fd83b003a Mon Sep 17 00:00:00 2001
From: Juan Yu <jyu@cloudera.com>
Date: Tue, 2 Sep 2014 18:10:51 -0700
Subject: [PATCH 142/596] MR1: CLOUDERA-BUILD, update jackson version to
 1.9.13

(cherry picked from commit da097ec56e0534454b907343fac9a060af49e25a)
(cherry picked from commit 101e94a7fee25674d62f6e15fa48bd0a202a1f2c)
---
 .../cloudera/maven-packaging/hadoop-core/pom.xml   |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-core/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-core/pom.xml
index 7e2b643..c7e6deb 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-core/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-core/pom.xml
@@ -129,12 +129,12 @@
     <dependency>
       <groupId>org.codehaus.jackson</groupId>
       <artifactId>jackson-mapper-asl</artifactId>
-      <version>1.5.2</version>
+      <version>1.9.13</version>
     </dependency>
     <dependency>
       <groupId>org.codehaus.jackson</groupId>
       <artifactId>jackson-core-asl</artifactId>
-      <version>1.5.2</version>
+      <version>1.9.13</version>
     </dependency>
     <dependency>
       <groupId>org.apache.zookeeper</groupId>
-- 
1.7.9.5

